app/javascript/controllers/visibility_controller.js

```diff
@@ -0,0 +1,29 @@
+import { Controller } from "stimulus"
+
```

Comment:
What do you think about renaming this to `dashboard_query_controller.js`? It started out as generic show/hide functionality, but it's pretty specific to the queries page now.

We'd need to update the reference to this controller in the view as well.